<?xml version="1.0"?>
<launch>
   <node name="rona_move_node" pkg="rona_move" type="rona_move_node">
      <!-- general -->
      <param name="pub_cmd_vel_topic"			type="string" value="cmd_vel"           />
      <param name="pub_state_topic"          type="string" value="rona/move/state"   />
      <param name="pub_process_topic"        type="string" value="rona/move/process" />
      <param name="sub_path_topic"           type="string" value="rona/move/path"    />
      <param name="sub_ctrl_topic"           type="string" value="rona/move/ctrl"    />
      <param name="map_frame"                type="string" value="map"               />
      <param name="robot_frame"              type="string" value="base_footprint"    />
      <param name="vel_lin_max"              type="double" value="0.4"               />
      <param name="vel_ang_max"              type="double" value="0.8"               />
      <param name="target_radius"            type="double" value="0.24"              />
      <param name="target_radius_final"      type="double" value="0.1"               />
      <param name="ang_reached_range"        type="double" value="0.05"              />
      <param name="lin_end_approach"         type="double" value="1.0"               />
      <param name="lin_ctrl_scale"           type="double" value="2.0"               />
      <param name="ang_ctrl_scale"           type="double" value="4.0"               />
      <param name="min_vel_value"            type="double" value="0.001"             />
      <param name="robot_reverse_frame"	     type="string" value="reverse"           />
      <param name="tf_stamp_offset"          type="double" value="0.0"             />
      <!-- omni -->
      <param name= "hold_pos"                type="bool"   value="true"              />
      <param name= "wait_for_rotation"       type="double" value="1.0"              />

      <!-- differential -->
      <param name="cos_pwr_n"                type="int"    value="4 "                />
      <param name="cos_fac_n"                type="double" value="0.7"               />
      <param name="do_endrotate"             type="bool"   value="false" 				 />

   </node>
</launch>
